swim/join_sender.go

```diff
@@ -162,6 +203,15 @@ func (j *joinSender) parseHosts(hostports []string) {
 	}
 }
 
+func newDelayOpts() *delayOpts {
+	return &delayOpts{
```

Comment:
got it. this code is going to undergo some revisioning after you and @CorgiMan gave me some ideas. So stay tuned.